apps/web/src/services/report.ts

```diff
@@ -311,7 +311,8 @@ export const createReportWorkbook = async (filter: PeriodFilter): Promise<WorkBo
     });
 
     return workbook;
-  } catch (e) {
+  } catch (e: any) {
+    notifyError(e.message || 'It has failed to create a report');
```

Comment:
what's 'It'? maybe we could change to like 'Unable to create a report' or 'There was an issue while attempting to create a report'